.github/workflows/upload-benchmarks.yml

```diff
@@ -0,0 +1,75 @@
+name: Upload-benchmarks
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
if we just want to do it on merge to develop, we should remove this line no? We should not do that on each PR?

Reply:
That's just for testing---I'm actually going to change it to nightlly